scripts/set_assignees.py

```diff
@@ -90,30 +91,114 @@ def process_pr(gh, maintainer_file, number):
             continue
 
         all_areas.update(areas)
-        is_instance = False
+        #
+        # DJL: Don't understand what "is_instance" is trying to record?
+        # It mucks with the weighting though.
+        #
+        #is_instance = False
         sorted_areas = sorted(areas, key=lambda x: 'Platform' in x.name, reverse=True)
         for area in sorted_areas:
-            c = 1 if not is_instance else 0
+            #c = 1 if not is_instance else 0
+
+            area_counter[area] += 1
 
-            area_counter[area] += c
-            labels.update(area.labels)
             # FIXME: Here we count the same file multiple times if it exists in
             # multiple areas with same maintainer
             for area_maintainer in area.maintainers:
-                found_maintainers[area_maintainer] += c
+                found_maintainers[area_maintainer] += 1
 
-            if 'Platform' in area.name:
-                is_instance = True
+            #if 'Platform' in area.name:
+            #    is_instance = True
```

Comment:
@nashif why do we discount platform when selecting an assignee?

Reply:
In cases where platforms code is changes, those are considered instances and depending on what other changes have been made, priority goes to the other areas, i.e. if you change i2c header and i2c driver of a platform, the i2c maintainer should be selected.

Reply:
But I've seen instances where 12 files are touched. 11 of them are NXP files and one was something else so the assignee when to the "1" something else.